File: server/nobody/src/main/java/info/chenliang/moba/message/MatchToBattleDispatcher.java
package info.chenliang.moba.message;
import java.io.*;
import info.chenliang.talky.*;
public class MatchToBattleDispatcher {
public static void dispatch(DataInputStream dis, MatchToBattleProxy proxy) throws Exception{
int fid = (int)(dis.read()&0xFFFF);
switch(fid)
{
}
}
}

File: client/Assets/Scripts/Message/info/chenliang/moba/message/MatchToBattleDispatcher.cs
namespace info.chenliang.moba.message{
using System;
using System.IO;
using MiscUtil;
using MiscUtil.IO;
using MiscUtil.Conversion;
using Talky;
public class MatchToBattleDispatcher {
public static void Dispatch(MemoryEndianBinaryReader reader, MatchToBattleProxy proxy) {
int fid = reader.ReadByte();
switch(fid)
{
}
}
}
}
